scaffold/package.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
Just wondering: do we want to include a `package-lock.json` file in the `scaffold/` directory, to lock the dependencies to a set of known-working versions?

Reply:
Are we not okay here as it runs `npm install` as part of the init stage?

Reply:
we're only allowing newer minor or patch level version with caret so no breaking changes will get through.